### PR TITLE
fix some methods.

### PR DESCRIPTION
**Pull request by cimadai**
- Change method name to camel case.
- Add created response in Add account/currency.
- Add domain URI patter.

I tested these code in 
https://github.com/hyperledger/iroha-scala/commit/3cfea94e893b73449a6d8c750d23b77b969ddeab

### CODE REVIEW

File: design/pattern.go [hunk 1]
```diff
@@ -4,7 +4,7 @@ const (
 	patternBase64      = `^[0-9a-zA-Z+/=]+$`
 	patternTimestamp   = `^[0-9]{1,18}$`
 	patternURLEncoded  = `^[0-9a-zA-Z-_.~]+$`
-	patternAccountUUID = `^[0-9a-f]{32}$`
-	patternDomainURI   = `^$` // TODO
+	patternAccountUUID = `^[0-9a-f-]{32}$`
+	patternDomainURI   = `^(?:[0-9a-zA-Z-._~]|%[0-9][0-9])+$` // following: RFC3986
```
Comment (upamune) [inline design/pattern.go:8]: DomainURI removes from the new IrohaObject and DomainUUID is added. But, as a URI this is a correct URI pattern. 👍

File: design/pattern.go [hunk 2]
```diff
@@ -4,7 +4,7 @@ const (
 	patternBase64      = `^[0-9a-zA-Z+/=]+$`
 	patternTimestamp   = `^[0-9]{1,18}$`
 	patternURLEncoded  = `^[0-9a-zA-Z-_.~]+$`
-	patternAccountUUID = `^[0-9a-f]{32}$`
-	patternDomainURI   = `^$` // TODO
+	patternAccountUUID = `^[0-9a-f-]{32}$`
```
Comment (upamune) [inline design/pattern.go:7]: Thanks 😄

File: design/api.go [hunk 3]
```diff
@@ -1,11 +1,21 @@
...
 import (
+	. "github.com/goadesign/goa/design"
 	. "github.com/goadesign/goa/design/apidsl"
 )
 
 var _ = API("Iroha-Gateway Server", func() {
 	Title("The Iroha API")
 	Host("localhost:8080")
 	Scheme("http")
+	ResponseTemplate(Created, func(pattern string) {
```
Comment (upamune) [inline design/api.go]: Is this necessary?

Comment (upamune) [reply design/api.go]: @cimadai

**Comment by upamune** [comment]
> Change method name to camel case.

Thanks ! 👍

**Comment by cimadai** [comment]
I'm sorry, actually `ResponseTemplate(Created, func(pattern string) {` is not necessary.

**Comment by upamune** [comment]
OK. Thanks.

**Comment by upamune** [comment]
Merged 👍

**Pull request merged**